docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,250 @@
+      roomActions.get();
+    };
+```
+
+##### <a id="owneeClassProps-nearestOwnerProps"></a>Property - `owneeClassProps.nearestOwnerProps` [#](#owneeClassProps-nearestOwnerProps)
+Allows the `Ownee` to access the `nearestOwnerProps` on the closest `OwnerClass`.
+nearestOwnerProps - should be static, if you need to pass in dynamic props use standard method (works well for an example)
+
+**NOTE** `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+##### <a id="owneeClassProps-nearestOwnerPropTypes"></a>Property - `owneeClassProps.nearestOwnerPropTypes` - type: OBJECT - optional [#](#owneeClassProps-nearestOwnerPropTypes)
+Allows you to perform prop validation on the `nearestOwnerProps` to this `OwneeClass` component. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.nearestOwnerPropTypes = {
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired,
+    };
+```
+
+##### <a id="owneeClassProps-anyPropTypes"></a>Property - `owneeClassProps.anyPropTypes` - type: OBJECT - optional [#](#owneeClassProps-anyPropTypes)
+Allows you to perform prop validation on both the standard props of a component, as well as those props available via `nearestOwnerProps`. For the full list of available properties that can be checked, please look at the [Reusable Components](http://facebook.github.io/react/docs/reusable-components.html) documenation.
+
+```javascript
+    owneeClassProps.anyPropTypes = {
+      message: React.PropTypes.object.isRequired,
+      deleteMessage: React.PropTypes.function.isRequired,
+      editMessage: React.PropTypes.function.isRequired
+    };
+```
+
+**NOTE** When performing prop validations with `anyPropTypes` that props in `this.props` are validated first, so they will override the information within nearestOwnerProps.
+
+#### <a id="createOwneeClass-OwneeClass"></a>Return - `OwneeClass` - type: OBJECT [#](#createOwneeClass-OwneeClass)
+The `createOwneeClass` method returns a `TuxOwneeClass` which is an extension on top of a `React` Class. You can then simply use it in render methods as you would any other `React` Class.
+
+***
+
+##### <a id="TuxOwneeClass-Guidelines"></a>4) TuxOwneeClass Guidelines [#](#TuxOwneeClass-Guidelines)
+It is quite possible to never take advantage of the versatility that is provided by using the `nearestsOwnerProps`. Yet you would be missing out on the benefits that it provides. Passing properties down through a multitude of components is error prone, as it is easy to forget to pass all properties or methods that you need. Part of this difficulty can be alleviated by using `propTypes` to validate a components prop inputs.
```

Comment:
Add on to last sentence: and using the `React Spread` syntax to cascade down properties. However, even with those tools prop sharing is difficult and fragile. Additionally, it significantly hurts our ability to reuse components since our components now have to request `props` that they would not normally need just so that they can share those `props` with their children.